src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -1893,6 +1893,9 @@ bool Matcher::match_rule_supported(int opcode) {
     case Op_CountPositives:
       return UseRVV;
 
+    case Op_CompressBits:
+      return UseRVV && (MaxVectorSize >= 16);
```

Comment:
Isn't it guaranteed that `MaxVectorSize >= 16` if `UseRVV` is true?

Reply:
After https://github.com/openjdk/jdk/pull/16498, it should be guaranteed `MaxVectorSize >= 16`.
Let me remove this condition after pr #16498 is pushed.